modules/app/dapr-containerapps-environment/daprComponent-sb.bicep

```diff
@@ -0,0 +1,66 @@
+param componentName string
+param location string
+param entityName string
+param containerAppEnvName string
+
+param createAzureServiceForComponent bool = true
+
+param scopes array = []
+
+var daprComponent = 'pubsub.azure.servicebus'
+var serviceBusConnectionStringName = 'sb-root-connectionstring'
+
+resource containerAppEnv 'Microsoft.App/managedEnvironments@2022-01-01-preview' existing = {
+  name: containerAppEnvName
+}
+
+resource daprServiceBus 'Microsoft.App/managedEnvironments/daprComponents@2022-03-01' = {
+  name: componentName
+  parent: containerAppEnv
+  properties: {
+    componentType: daprComponent
+    version: 'v1'
+    secrets: [
+      {
+        name: serviceBusConnectionStringName
+        value: '${listKeys('${servicebus.id}/AuthorizationRules/RootManageSharedAccessKey', servicebus.apiVersion).primaryConnectionString};EntityPath=${entityName}'
```

Comment:
not critical, but it might make sense to get an `existing` reference to the `RootManageSharedAccessKey` resource and then accessing the member fuction on the symbolic names (i.e. `authRule.listKeys().primaryConnectionString`)